Classes/Repository/RepositoryIssuesViewController.swift

```diff
@@ -85,6 +85,12 @@ IndicatorInfoProvider {
             case .error:
                 self?.error(animated: trueUnlessReduceMotionEnabled)
             case .success(let payload):
+                if let itemCount = payload.numberOfItems,
```

Comment:
It might be worth creating a `strongSelf` here (`guard let strongSelf = self else { return }`) so we don't have to deal with the optional `self`. What do you think?

Reply:
Why not just `guard let self = self else { return }`? 😄

Reply:
Also possible 😊